dubbo-remoting/dubbo-remoting-http12/src/main/java/org/apache/dubbo/remoting/http12/message/LengthFieldStreamingDecoder.java

```diff
@@ -20,7 +20,6 @@
 import org.apache.dubbo.remoting.http12.exception.DecodeException;
 
 import java.io.ByteArrayInputStream;
-import java.io.ByteArrayOutputStream;
 import java.io.IOException;
```

Comment:
**Performance Issue**: Unused import 'java.io.ByteArrayOutputStream' should be removed to keep code clean. <br> **Fix**: Remove the unused import statement to clean up the code. <br> **Code Suggestion**: 
 ```
 -import java.io.ByteArrayOutputStream;
 ```